### ISSUE TITLE
wp_get_sites is depreciated

### ISSUE THREAD

**Issue by mrbm** [J-Rey/wp-cron.py#2]
I think the new one is get_sites()

refer to this: https://developer.wordpress.org/reference/functions/wp_get_sites/

**Comment by J-Rey** [comment]
Yep, already attempted to be patched upstream: https://github.com/pressjitsu/wp-cron.py/pull/1
